module/testmodule/main.go

```diff
@@ -106,9 +107,21 @@ func mainWithArgs(ctx context.Context, args []string, logger logging.Logger) err
 func newHelper(
 	ctx context.Context, deps resource.Dependencies, conf resource.Config, logger logging.Logger,
 ) (resource.Resource, error) {
+	var dependsOnSensor sensor.Sensor
```

Comment:
[opt] Doesn't really matter, but generally I've seen [`FromDependencies`](https://github.com/viamrobotics/rdk/blob/main/components/sensor/sensor.go#L48) used instead of `for` loop searches like the one you're doing.

Reply:
Cool; yeah it's still awkward, sorry about that. What you have seems fine. I think you're right that if we wanted this to "look good" (and I don't really care too much since this is just testing code) we'd need a type for helper configs and probably an associated `Validate` method.